ingress/src/main/java/com/redhat/developer/ingress/api/exceptions/IngressRuntimeException.java

```diff
@@ -0,0 +1,19 @@
+package com.redhat.developer.ingress.api.exceptions;
+
+import javax.ws.rs.core.Response;
+
+public class IngressRuntimeException extends IngressException {
```

Comment:
Do we need this? `IngressException` is already a `RuntimeException`.

Reply:
removed